assembl/static2/tests/unit/components/debate/thread/__snapshots__/answerForm.spec.jsx.snap

```diff
@@ -102,7 +102,7 @@ exports[`AnswerForm component should render a answer form 1`] = `
             }
           }
           handleInputFocus={[Function]}
-          maxLength={3000}
+          maxLength={0}
```

Comment:
`NO_BODY_LENGTH` instead. Same thing, but let's use the constants we've created.

Reply:
mmh I didn't notice there was a defaultProps to 0, weird choice. Anyway, this is going